src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
+
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login("bob", "1234ss");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("User login attempt failed", e);
```

Comment:
I would prefer warn level here, since error is mainly used when you have some kind of error in app work. Auth exception is usually just a mistype from user